[2025-04-06T07:40:01.001+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T07:30:00+00:00 [queued]>
[2025-04-06T07:40:01.009+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T07:30:00+00:00 [queued]>
[2025-04-06T07:40:01.010+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-06T07:40:01.025+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_data_loading> on 2025-04-06 07:30:00+00:00
[2025-04-06T07:40:01.032+0000] {standard_task_runner.py:60} INFO - Started process 212 to run task
[2025-04-06T07:40:01.035+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_file_upload_dag', 'run_data_loading', 'scheduled__2025-04-06T07:30:00+00:00', '--job-id', '197', '--raw', '--subdir', 'DAGS_FOLDER/gcs_fileupload_dag.py', '--cfg-path', '/tmp/tmptzq0bdib']
[2025-04-06T07:40:01.037+0000] {standard_task_runner.py:88} INFO - Job 197: Subtask run_data_loading
[2025-04-06T07:40:01.056+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-06T07:40:01.099+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_file_upload_dag.run_data_loading scheduled__2025-04-06T07:30:00+00:00 [running]> on host bb17cde9267a
[2025-04-06T07:40:01.180+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_file_upload_dag' AIRFLOW_CTX_TASK_ID='run_data_loading' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T07:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-06T07:30:00+00:00'
[2025-04-06T07:40:01.181+0000] {gcs_fileupload_dag.py:16} INFO - Calling data_loading from DAG...
[2025-04-06T07:40:01.182+0000] {logging_mixin.py:188} INFO - Starting data_loading...
[2025-04-06T07:40:01.476+0000] {logging_mixin.py:188} INFO - GitHub API rate limit: 5000/5000
[2025-04-06T07:40:02.258+0000] {logging_mixin.py:188} INFO - Repository Ai-app-by-using-python already cloned. Skipping.
[2025-04-06T07:40:02.259+0000] {logging_mixin.py:188} INFO - Repository MaxBrowser already cloned. Skipping.
[2025-04-06T07:40:02.260+0000] {logging_mixin.py:188} INFO - Repository portfolio already cloned. Skipping.
[2025-04-06T07:40:02.260+0000] {logging_mixin.py:188} INFO - Repository learning-python already cloned. Skipping.
[2025-04-06T07:40:02.261+0000] {logging_mixin.py:188} INFO - Repository ToDo-List already cloned. Skipping.
[2025-04-06T07:40:02.261+0000] {logging_mixin.py:188} INFO - Data loading completed.
[2025-04-06T07:40:02.263+0000] {gcs_fileupload_dag.py:18} INFO - Done running data_loading.
[2025-04-06T07:40:02.264+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-06T07:40:02.276+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_file_upload_dag, task_id=run_data_loading, execution_date=20250406T073000, start_date=20250406T074001, end_date=20250406T074002
[2025-04-06T07:40:02.332+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-06T07:40:02.356+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
